### PR TITLE
Best practices

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
## You can find all the files related to the to do list

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <div class="container__footer">
+                <button id="buttonDelete" class="delete-button" type="button">Clear all done</button>              
+            </div>
+        </div>
+       
+    </main>
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400&family=Raleway:wght@200;300;500;600&display=swap" rel="stylesheet">
```
Comment (Peter1907) [inline src/index.html:31]: I think you should use the head when linking CSS or fonts ... It's not fatal but it's a practice recommended by MDN ... take a look here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

File: src/modules/activities.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+  #addToState(object) {
+    this.#activitiesArr.push(object);
+    this.#saveLocal();
+  }
+
+  #deleteFromState(index) {
+    this.#activitiesArr.splice(index, 1);
+    this.#saveLocal();
+    console.log(this.#activitiesArr);
```
Comment (Peter1907) [inline src/modules/activities.js:41]: It's best not to leave any (console.logs)s in your code.

**Review by Peter1907 — CHANGES_REQUESTED** [review]
14d82a49c68b30cda311f06cb49d3b71

You did a really great job on your structure of the HTML, CSS & JavaScript, there are only a few issues that you might wanna address for a better structure of your project.

Please read the comments below.
For any questions, you can leave me a comment & make sure to mention me.
HAPPY <coding>

File: src/modules/activities.js (hunk 2 above)
Comment (LuisDavidRodriguez) [reply src/modules/activities.js:41]: Yeah i will prune all of them thank you dear reviewer!

File: src/index.html (hunk 1 above)
Comment (LuisDavidRodriguez) [reply src/index.html:31]: Ok great! thank you

**Pull request merged**